pipelines/managed/release-to-github/README.md

```diff
@@ -24,6 +24,7 @@ Tekton release pipeline to release binaries extracted from the image built with
 ## Changes in 4.6.0
 * Pass taskGitUrl and taskGitRevision to extract-binaries-from-image task
 * Pass taskGitUrl and taskGitRevision to base64-encode-checksum task
+* Pass taskGitUrl and taskGitRevision to collect-gh-params task
```

Comment:
This should be a new version, right? Not that I care much about versions, but we should probably stay consistent until we remove them (soon).